ssl/internal.h

```diff
@@ -1988,6 +2029,8 @@ struct SSL_HANDSHAKE {
   // scts_requested is true if the SCT extension is in the ClientHello.
   bool scts_requested : 1;
 
+  bool received_custom_extension:1;
```

Comment:
nit: spacing of field size isn't consistent with the surrounding values.  E.g. `: 1;`